src/test/java/io/vertx/ext/asyncsql/TenThousandRowsData.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright 2015 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.ext.asyncsql;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class TenThousandRowsData {
+  public static final Map<Integer, String> TABLE = new TreeMap<>();
+
+  static {
+    int i = 0;
+    while( i < 10000) {
```

Comment:
Please check the formatting of this file, below, there are also `.append` calls that should probably be 2 spaces